README.md

```diff
@@ -46,6 +34,21 @@ Download and run the `smartthings.msi` installer from the [latest Release](https
 1. Download the appropriate zipped binary from the [latest Release](https://github.com/SmartThingsCommunity/smartthings-cli/releases).
 1. Extract and install it on your system path. It does not need administrator privileges but will need to be executable.
 
+### Other
+
+The CLI can be used on any platform that supports Node 24.
+
+First, install at least [Node version 24.8.0](https://nodejs.org/en/download/current) on your machine.
```

Comment:
This link will likely change to something else in the near future rather than node 24.  Do you want to just say latest Node or did you need it to say 24 specifically?  If so maybe there is a direct link to node24 that can be used.